### ISSUE TITLE
Move WASC TC v2 URLs to vulndb/data

### ISSUE THREAD

**Issue by robocoder** [vulndb/data#35]
To avoid duplication:
- https://github.com/vulndb/python-sdk/blob/master/vulndb/constants/wasc.py
- https://github.com/vipsoft/vulndb-php/blob/master/src/Service/ReferenceService.php#L55-L104

**Comment by andresriancho** [comment]
I agree, this should be a JSON file somewhere in our data repository and then the different libs should consume it. My proposal is to add a `meta` directory inside the `db` directory and inside add this data.

**Comment by andresriancho** [comment]
This duplication was discovered at https://github.com/vulndb/data/issues/33 and affects python, go and php sdks.

**Comment by andresriancho** [comment]
## TODO
- [ ] Find other potential sources of code duplication
- [ ] Document how they'll be implemented in `data`
- [ ] Move them to `data` and change the SDKs
